vuepress/docs/next/docs/consume/identity-management.md

```diff
@@ -1,53 +1,48 @@
 # Entando Identity Management -- Keycloak
 
-Entando Identity Management is powered by Keycloak. The Keycloak instance used for your Entando apps
-can be [externally installed](../../tutorials/devops/external-id-management) or you can use a dedicated instance on a per application basis. The sections below details the architecture and documentation required to customize your Keycloak instance.
+Entando Identity Management authenticates Entando Applications with Keycloak. The Keycloak instance can be either [externally installed](../../tutorials/devops/external-id-management) or dedicated on a per-application basis. The architecture and requirements to customize your Keycloak instance are described below.
```

Comment:
Some of the original language was a bit clearer "Entando Identity Management is powered by Keycloak" or "Entando Identity Management is based on open source Keycloak." It's too limiting to say it just manages authentication (verifying identity) since authorization (providing access rights based on that identity) is also a big part of Keycloak's use.

"dedicated on a per-application basis" -> "use a dedicated instance on a per application basis" or "installed just for a specific application" could be better and simpler.